src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -443,11 +443,15 @@ When using the `<ViewTransitions />` router, the following steps occur to produc
 7. The router executes any new scripts added to the page.
 8. The `astro:page-load` event fires. This is the end of the navigation process.
 
-## Script behavior during page navigation
+## Script behavior
 
 When navigating between pages with the `<ViewTransitions />` component, scripts are run in sequential order to match browser behavior. 
 
-If you have code that sets up global state, this state will need to take into account that the script might execute more than once. Check for the global state in your `<script>` tag, and conditionally execute your code where possible:
+Bundled module scripts, which are the default scripts in Astro, are only ever executed once. If a bundled module script exists on pages that are transitioned to after its initial execution they will be ignored. Inline scripts, ones with `is:inline` attribute, will not re-execute if they exist on both the page being transitioned to and the one the user is currently on. 
```

Comment:
```suggestion
### Script re-execution

[Bundled module scripts](/en/guides/client-side-scripts/#script-processing), which are the default scripts in Astro, are only ever executed once. After initial execution they will be ignored, even if the script exists on the new page after a transition.
```
It seems very confusing to have the sentence about inline scripts running once, then immediately saying that they *can* run more than once in a following paragraph. I think keeping inline stuff to its own paragraph makes sense.